src/utils/validateIdentifierInputs.test.ts

```diff
@@ -41,6 +41,37 @@ describe("validateIdentifierInputs", () => {
     ).toBe(true);
   });
 
+  it("should throw error if input is phone number and fails default regex", () => {
```

Comment:
There is already a test below on line 121 leh

Reply:
This is rebased code, nothing to do with this PR!